### PR TITLE
17.0 mig base unece

### PR DESCRIPTION
**Pull request by youelEl**

### CODE REVIEW

File: base_unece/models/unece_code_list.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    _sql_constraints = [
+        (
+            "type_code_uniq",
+            "unique(type, code)",
+            "An UNECE code of the same type already exists",
+        )
+    ]
+
+    @api.depends("code", "name")
+    def name_get(self):
```
Comment (alexis-via) [inline base_unece/models/unece_code_list.py:33]: For v17, you should replace name_get() by _compute_display_name()

File: base_unece/security/ir.model.access.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_unece_code_list_read,Read access on unece.code.list to everybody,model_unece_code_list,,1,0,0,0
```
Comment (alexis-via) [inline base_unece/security/ir.model.access.csv:2]: there is a warning caused by missing group, no ?
Take inspiration on the ACL of res.country in the base module: they now have 3 read ACLs: base.group_user, base.group_portal and base.group_public

Comment (youelEl) [reply base_unece/security/ir.model.access.csv:2]: ok @alexis-via, I will take that into consideration

**Review by gurneyalex — APPROVED** [review]

**Comment by alexis-via** [comment]
I implemeted the remarks of the reviewers in this PR https://github.com/OCA/community-data-files/pull/193

**Pull request closed without merging**